ParcialProgramacion4/src/Ejecutora.java

```diff
@@ -214,13 +214,13 @@ public static void menuUsuario() {
 	                
 	            break;
 	        case 4:
-	                
+	        	empresa.agregarHabilidadEmpleado();    
 	            break;
 	        case 5:
-	                
+	        	empresa.editarAniosEmpleado();    
```

Comment:
LLegamos a la convencion de que la enie se escribe nn
```suggestion
	        	empresa.editarAnniosEmpleado();    
```

Reply:
listo